test/AspNet.Security.OAuth.Providers.Tests/Infrastructure/ApplicationFactory.cs

```diff
@@ -92,7 +92,7 @@ private static void ConfigureApplication<TOptions>(IApplicationBuilder app, OAut
            .UseEndpoints(endpoints =>
            {
                endpoints.MapGet(
-                   "/me",
+                   "/userinfo",
```

Comment:
Please revert this change, it is nothing specific to LinkedIn, the fact it is the same as their old URL is a coincidence.